docs/pages/database-access/reference/labels.mdx

```diff
@@ -12,14 +12,24 @@ of the following values:
 | - | - |
 | `cloud`   | database resources created by auto-discovery. |
 | `config`  | database resources manually defined in the `database_service.databases` section of `teleport.yaml`. |
-| `dynamic` | database resources created with "tctl create" command or web interface. |
+| `dynamic` | database resources created through [dynamic registration](../guides/dynamic-registration.mdx) like `tcl create` command. |
 
 ## Auto-discovery
 
 The labels of auto-discovered databases primarily come from the tags that are
 assigned to the original cloud resources, such as the resources tags of an
 Amazon RDS instance. 
 
+The following tags will override Teleport's default behavior if assigned to the
+original cloud resources:
+
+| Tag name | Description |
+| - | - |
+| `TeleportDatabaseName`                   | Overrides the name of the discovered database. |
+| `teleport.dev/database_name`             | (AWS only) Overrides the name of the discovered database. |
```

Comment:
Should we not mention this one anymore? If we do, we should note that the aws specific db name override label takes lower precedence than `TeleportDatabaseName`